manifests/init.pp

```diff
@@ -117,12 +133,16 @@
 # @author Trevor Vaughan <tvaughan@onyxpoint.com>
 #
 class simp (
+  # This parameter set from data in modules
   Hash                            $scenario_map,
-  Enum['simp',
-       'simp_lite',
-       'poss',
-       'none',
-       'remote_access']           $scenario                   = 'simp',
+  Enum[
```

Comment:
@heliocentric now that there is actual documentation about the scenarios, this can be changed. We'll need a function created to snag all of the scenarios and add them as the validation list though. They shouldn't be able to input something that is invalid. That can be a new ticket.